app/experimenter/experiments/migrations/0212_alter_nimbusexperiment_languages_field_for_mobile_client.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+from experimenter.base.models import Language
+from experimenter.experiments.models import NimbusExperiment
+
+
+def update_languages_field_for_mobile_client(apps, schema_editor):
+    NimbusExperiments = apps.get_model("experiments", "NimbusExperiment")
+
+    for experiment in NimbusExperiments.objects.all():
+        if experiment.application != NimbusExperiment.Application.DESKTOP:
+            for locale in experiment.locales.all():
+                locale_code = locale.code[:2]
+                language = Language.objects.filter(code=locale_code).first()
+                if language:
+
+                    experiment.languages.add(language.id)
+            experiment.locales.clear()
+            experiment.save()
```

Comment:
I think `save` is redundant here since both `locales` and `languages` are related entities so the `add` and `clear` calls both already hit the database.  `save` will only persist changes to local fields right on the instance.